packages/editor/src/plugins/dropzone-image/index.ts

```diff
@@ -0,0 +1,70 @@
+import { EditorPluginType } from '@editor/types/editor-plugin-type'
+
+import { DropzoneImageEditor } from './components/editor/editor'
+import { BackgroundShape, DropzoneVisibility } from './types'
+import {
+  type EditorPlugin,
+  type EditorPluginProps,
+  object,
+  string,
+  child,
+  number,
+  list,
+  optional,
+} from '../../plugin'
+
+export const positionData = object({
+  top: number(0),
+  left: number(0),
+})
```

Comment:
Is this relative or absolute positioning? I would argue for relative positioning...

Reply:
This is for the functionality of the canvas.

Reply:
I'm not sure why this is an important detail can you please explain?

The container is relative and the answerZones are absolute

Reply:
`packages/editor/src/plugins/dropzone-image/components/answer-zone/answer-zone.tsx:76`

Reply:
`packages/editor/src/plugins/dropzone-image/components/editor/editor-canvas.tsx:117`

Reply:
> I'm not sure why this is an important detail can you please explain?

Let's say you upload an image in 800x600 dimension. However due to a smaller screen the image is resized and shown in 400x300. Then the absolute positions of the drag & drop zones are messed up. This is relevant for showing the plugin in mobile devices.

The easiest is to store relative positions (top: 0-100% and left: 0-100%) and to position the drag & drop zones based on the current size of the image.

Reply:
Edit mode is desktop only, working on the fix for static view now

Reply:
Static view is mobile ready now.

I would call the screen changing size, and responsiveness to that, out of scope for this iteration

Reply:
> I would call the screen changing size, and responsiveness to that, out of scope for this iteration

Meaning that we would do it later or never?